tests/auth.test.js

```diff
@@ -0,0 +1,53 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../src/index';
+import messages from '../src/utils/messages';
+import codes from '../src/utils/codes';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('User sign up', () => {
+  it('Should return 201', (done) => {
+    chai
+      .request(server)
+      .post('/api/auth/signup')
+      .send({
+        firstname: 'John',
```

Comment:
Can you create a `data` folder inside the test folder, create a file called `mockData` that returns an array of objects, then you can move this object inside that array and call it here. like `mockData[0]`